### DISCUSSION TITLE
Custom cluster domain

### DISCUSSION THREAD

**Discussion by jurim76** [rabbitmq/cluster-operator#1543] [General]
Is it possible to override default node domain "cluster.local" with custom one (for example "test.cluster.local") in rabbitmq cluster?

**Comment by mkuratczyk** [comment]
Do you mean this?
https://github.com/rabbitmq/cluster-operator/blob/main/docs/examples/ipv6/rabbitmq.yaml#L16
https://www.rabbitmq.com/cluster-formation.html#peer-discovery-k8s

If not, please be more specific.

**Reply by jurim76** [reply]
Thank you!
